services/api/src/typeDefs.js

```diff
@@ -562,6 +577,10 @@ const typeDefs = gql`
     Returns the costs for all billing groups
     """
     allBillingGroupsCost(month: String!): JSON
+    """
+    Returns the Billing Group Modifiers for a given Billing Group (all if no month filter defined)
+    """
+    billingModifiers(input: GroupInput!, month: String): [BillingModifier]
```

Comment:
I think the naming convention here would be `allBillingModifiers`

Reply:
Addressed in 82c1c2134ef0c3f6097293faabbf95ce44e64346